[All Hosts] (manifest) update requirements-related docs (#3190)
OfficeDev/office-js-docs-pr: docs/reference/manifest/sets.md

---
title: Sets element in the manifest file
description: The Sets element specifies the minimum set of Office JavaScript API your Office Add-in requires in order to be activated by Office or to override base manifest settings.
ms.date: 01/22/2022
ms.localizationpriority: medium
---

# Sets element

> The meaning of this element depends on where it is used in the manifest.

Reviewer: ```suggestion
The meaning of this element depends on where it's used in the manifest.
```

## In the base manifest

When used in the base manifest, that is, the parent **Requirements** element is a direct child of [OfficeApp](officeapp.md), the **Sets** element specifies the minimum subset of the Office JavaScript API requirements ([requirement sets](../../develop/office-versions-and-requirement-sets.md#specify-office-applications-and-requirement-sets) that your Office Add-in needs in order to be activated by Office.

**Add-in type:** Content, Task pane, Mail

## As a grandchild of a VersionOverrides element

> Specifies the minimum set of Office JavaScript API requirements ([requirement sets](../../develop/office-versions-and-requirement-sets.md#specify-office-applications-and-requirement-sets) that must be supported by the Office version and platform (such as Windows, Mac, iOS, and web) in order for the [VersionOverrides](versionoverrides.md) to take effect.

Reviewer 2: ```suggestion
Specifies the minimum set of Office JavaScript API requirements ([requirement sets](../../develop/office-versions-and-requirement-sets.md#specify-office-applications-and-requirement-sets)) that must be supported by the Office version and platform (such as Windows, Mac, web, and iOS or iPad) in order for the [VersionOverrides](versionoverrides.md) to take effect.
```
